convert2rhel/grub.py

```diff
@@ -572,6 +581,31 @@ def post_ponr_set_efi_configuration():
         _log_critical_error(e.message)
 
 
+def update_grub_after_conversion():
+    """Update grub images and config after conversion."""
```

Comment:
```suggestion
    """Update GRUB2 images and config after conversion.
    
    This is mainly a protective measure to prevent issues in case the original distribution GRUB tooling
    generates images that expect different format of a config file. To be on the safe side we
    rather re-generate both the GRUB config file and the GRUB image.

    We opted for doing that only for GRUB2 and not for GRUB Legacy intentionally for the reason of
    RHEL 6 having transitioned to the Extended Life-cycle Support phase.
    """
```